### DISCUSSION TITLE
Voice announcement is too frequent

### DISCUSSION THREAD

**Discussion by anten1222** [osmandapp/OsmAnd#12545] [Q&A] [answered]
Voice announcement is too frequent
 Profile bike
 The announcement to turn comes 500 meters before the turn and then another 3 times
 I would like to be able to set how many meters before the announcement is made.
 Route recalculated, is it possible to turn off this announcement completely?

Sprachansage ist zu heufig 
Profil Fahrrad 
Die Ansage zum abbiegen kommt schon 500 Meter vor dem abbiegen und dann noch 3mal
Ich hätte gerne die Möglichkeit einzustellen ab wieviel Meter vorher die Ansage erfolgt.
Route neu berechnet, ist es möglich diese Ansage ganz abzustellen?

**Comment by sonora** [comment, marked as answer]
Here's the algorithm: https://docs.osmand.net/en/main@latest/development/algorithms/voice-prompt-triggering

It should be 319 meters? And there should only be a total of 3 prompts? Or are you not using the bicycle profile?

Have you tried the voice "German casual"? That one should skip the first announcement, as I recall, see #10150.

**Comment by pebogufi** [comment]
"German casual" is a very good improvement.
I translated parts of it to schwäbisch, and now driving makes me happy !

**Comment by anten1222** [comment]
I think if I could set the meters myself it would be optimal
 And yes, I use the bike profile

**Comment by sonora** [comment]
@anten1222 So what's your issue, exactly:
- If the prompts are too frequent for your needs, you can use e.g. voices like 'German casual' which suppress some prompts
- Or are they too early, and you need to find out how to shift them, or even generally request the bike profile's timing needs fine-tuning?

**Comment by anten1222** [comment]
My English is very bad I use Google translate 
For cycling, I use the Bicycle profile. I have set the average speed to 3km/h and the announcement on the last metres. The announcement is still too frequent. Furthermore, the announcement The route is being recalculated is too frequent. It depends on the GPS accuracy, if I could turn off this announcement completely it would be very nice.

**Comment by sonora** [comment]
I suggest 2 solutions for you:

- Please in OsmAnd's menu go to "Download maps" and and download, then use the German "German (casual)" TTS voice,. It is shorter and skips the first (~300 m) announcement.
- For the route recalculation, please go to the bicycle profile's "Navigation settings" and and increase the setting "Minimal distance to recalculate the route" from it's default 120 m to something higher.

I think that with these 2 configuration changes you may solve your issues?

**Comment by drp4positive** [comment]
Hey there.
I just started using OsmAnd. The voice guidance drives me crazy. I am using the standard driving profile and the guidance tells me EVERY SINGLE KILOMETER how far I stil have to go ont the "Autobahn" or road. Like "100km left." - "99 Kilometers left." That is total nonesense. I only need to know when I do have to make a turn a couple of kilometers in advance (like google maps does). I tried EVERYTHING in the iOs app to disable it, but nothing is working. Only solution is to disable the voice guidance (tried english and german). But that is not very helpful if you are driving from one end of Germany to the next.

Please tell me what I have missed to change the setting to annouce voice guidance only at important turnings.

Thank you

**Reply by vshcherb** [reply]
Actually you've probably enabled something like repeat voice announcement every minute, please check Navigation Settings for your profile.

**Comment by drp4positive** [comment]
Thanjs. I checked that option already in the beginning. It's not that, sadly. Would have been an easy solution. Any other ideas? Thx

**Comment by sonora** [comment]
Regarding the "Repeat navigation instruction'  setting, make sure you understand it is profile dependent, so disable it for the correct (or all) profile.

Another idea: Do you maybe have 'Accessibility mode'  selected?
